### ISSUE TITLE
VPAT 2.4.3, Tab Order

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#17]
Screengrab/videos (insert file names here)
1, Keyboard, items missing when tabbing.mp4

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Tab order does not match the visual layout.

Steps to reproduce
1.  0:07 The submenu cannot be activated in the expected order as previously reported.

2.  0:14  Tabbing from “Look up Permit” button moves to “Report and Issue” which is out of order, and then jumps to “Privacy Policy”

Desired Result
Keyboard tab order should match the visual order.

Actual Result
Keyboard tab order does not match the visual order.

[1, Keyboard, items missing when tabbing.zip](https://github.com/CityOfBoston/Iterators/files/3467665/1.Keyboard.items.missing.when.tabbing.zip)

**Comment by phillipbentonkelly** [comment]
@paylakatel this is now on Staging, you can check it out tomorrow. thx

**Comment by paylakatel** [comment]
This works great on staging - we can move into production. Thank you!

**Comment by phillipbentonkelly** [comment]
Deployed, Live; Leaving the ticket un-closed until Monday
